hs_core/tests/api/rest/test_custom_scimeta.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework import status
+
+from hs_core.hydroshare import resource
+
+from .base import HSRESTTestCase
+
+
+class TestCustomScimetaEndpoint(HSRESTTestCase):
+    def setUp(self):
+        super(TestCustomScimetaEndpoint, self).setUp()
+
+        self.rtype = 'GenericResource'
+        self.title = 'My Test resource'
+        res = resource.create_resource(self.rtype,
+                                       self.user,
+                                       self.title)
+
+        self.pid = res.short_id
+        self.resources_to_delete.append(self.pid)
+
+    def test_set_custom_metadata(self):
```

Comment:
It seems the difference between these 2 tests is that one passing 2 sets of key/value and the other one passing one set of key/value. In that case would it be better to rename the test function names as ```test_set_custom_metadata_multiple()``` and ```test_set_custom_metadata_single()```?